### PR TITLE
chore(northlight): improved Mobile Layout Experience

### PR DESCRIPTION
**Pull request by ilija-**
**northlight[dot]dev**

- Desktop and Mobile layout improvements
- A couple of other pages with small improvements
- Added a few missing icons
- Other minor layout fixes

https://github.com/mediatool/northlight/assets/5406237/be072e96-de24-4180-b53a-942f4d23629a

### CODE REVIEW

**Comment by ilija-** [comment]
still debugging for some layout anomalies and cleaning code ...

File: docs/src/app/sandbox.tsx [hunk 1]
```diff
@@ -1,154 +1,198 @@
...
+
+                  <Flex
+                    flex="1"
+                    pl={ { md: '21rem' } }
+                    pt={ 4 }
+                    boxSizing="border-box"
+                  >
+                    <VStack spacing={ 4 }>
+                      <Routing fallback={ head(routes)?.path } routes={ routes } />
+                    </VStack>
```
Comment (SebCodesTheWeb) [inline docs/src/app/sandbox.tsx:192]: If you try to go to https://deploy-preview-180--luminous-medovik-367cf4.netlify.app/tokens/global-tokens its gonna look really weird becuase the tokens jsoncrack does not take up entire width, to fix this put w="full" on `<Flex> `and `<VStack>` here

File: docs/src/app/components/nav-side-link.tsx [hunk 2]
```diff
@@ -16,8 +16,8 @@ interface CustomNavLinkProps {
 
 const getBgColor = (currentTheme: CurrentTheme) => {
   switch (currentTheme) {
-    case 'webappTheme': return 'blue.100'
-    case 'tottTheme': return 'blue.800'
+    case 'webappTheme': return 'background.button.ghost-hover'
+    case 'tottTheme': return 'background.button.ghost-hover'
     default: return 'bg.filled'
   }
 }
```
Comment (SebCodesTheWeb) [inline docs/src/app/components/nav-side-link.tsx:23]: You can probably remove this entire block of code and just set bgColorHover to background.button.ghost-hover directly

File: docs/src/app/sandbox.tsx [hunk 3]
```diff
@@ -1,154 +1,198 @@
...
+                  w={ { base: '100%', md: 'auto' } }
+                  maxW={ { base: '100vw', md: 'none' } }
+                  overflowX={ { base: 'hidden', md: 'initial' } }
+                >
+                  <Box
+                    w="20rem"
+                    h="100%"
+                    px={ 2 }
+                    overflowY="auto"
+                    position="fixed"
```
Comment (SebCodesTheWeb) [inline docs/src/app/sandbox.tsx:172]: This is some nitpicking, but you probably want to use the spacing and sizing tokens instead of px and rem values, e.g. pt="64px" => pt="16" and w="20rem" => w="80". You can find them under the tokens picker

File: docs/src/docs/app.tsx [hunk 4]
```diff
@@ -67,300 +66,384 @@ const markdownPreviewContent = changelogContent.reduce((acc, item, index) => {
...
+        >
+          <Icon
+            as={ NorthlightLogoXs }
+            boxSize={ { base: '6rem', md: '24rem' } }
+            alignSelf="center"
+            viewBox="0 -2 22 22"
+          />
+        </GridItem>
+        <GridItem order={ { base: 2, md: 1 } }>
+          <Box color="white">
```
Comment (SebCodesTheWeb) [inline docs/src/docs/app.tsx:131]: I think you can remove color="white", idk think it does anything because it is overriden by the children

File: docs/src/docs/app.tsx [hunk 5]
```diff
@@ -67,300 +66,384 @@ const markdownPreviewContent = changelogContent.reduce((acc, item, index) => {
...
+        justifyContent="center"
+        alignItems="center"
+        w="full"
+        px={ { base: 5 } }
+        boxSizing="border-box"
+      >
+        <SimpleGrid
+          columns={ { base: 1, md: 2, lg: 3 } }
+          spacingX="20px"
+          spacingY="20px"
```
Comment (SebCodesTheWeb) [inline docs/src/docs/app.tsx:216]: Could use tokens here instead

File: docs/src/docs/app.tsx [hunk 6]
```diff
@@ -374,389 +457,302 @@ const DocsApp = () => (
...
-            <H4>Developer Centered</H4>
-          </HStack>
-        </Box>
-      </HStack>
-    </Flex>
+      <Box
+        borderWidth="1px"
+        borderRadius="lg"
+        p={ 4 }
+        boxShadow="none"
```
Comment (SebCodesTheWeb) [inline docs/src/docs/app.tsx:611]: You can use borderWidth="xs" on all of these and downwards

**Review by SebCodesTheWeb — CHANGES_REQUESTED** [review]

**Comment by ilija-** [comment]
Hey hey @SebCodesTheWeb, w8 man, I'm not done yet i put it up to debug :D But thanks.

File: docs/src/docs/app.tsx [hunk 7]
```diff
@@ -67,303 +66,379 @@ const markdownPreviewContent = changelogContent.reduce((acc, item, index) => {
...
+            </HStack>
+          </Box>
+        </GridItem>
+      </Grid>
+    </Fade>
+    <Fade in={ true } enterDelay={ 500 }>
+      <Box px={ { base: 5 } }>
+        <Card p={ 10 } bgColor="bg.layer">
+          <CardHeader>
+            <Heading size="md">Navigate the celestial with Northlight 🚀✨</Heading>
```
Comment (SebCodesTheWeb) [inline docs/src/docs/app.tsx:179]: Maybe reword it like "celestial web", or "celestial realms"?

File: docs/src/app/sandbox.tsx [hunk 8]
```diff
@@ -1,154 +1,194 @@
...
+                        boxSize={ 16 }
+                        viewBox="0 0 20 18"
+                      />
+                      <Capitalized>North</Capitalized><Capitalized sx={ { color: 'brand' } }>Light</Capitalized>
+                    </Box>
+                  </Link>
+                  <Box
+                    flexGrow={ 1 }
+                    display={ { base: 'flex', md: 'flex' } }
+                    justifyContent="centar"
```
Comment (SebCodesTheWeb) [inline docs/src/app/sandbox.tsx:94]: centar?

**Review by SebCodesTheWeb — CHANGES_REQUESTED** [review]

**Review by SebCodesTheWeb — APPROVED** [review]
